packages/iTwinUI-react/src/core/Tile/Tile.tsx

```diff
@@ -73,6 +73,10 @@ export type TileProps = {
    * Dropdown menu containing `MenuItem`s.
    */
   moreOptions?: React.ReactNode[];
+  /**
+   * Status of the tile.
+   */
+  status?: 'positive' | 'warning' | 'negative';
```

Comment:
Does this PR also aim to implement the status and loading for Folder tiles?

![image](https://user-images.githubusercontent.com/45748283/197788204-6d3d81b0-fbb0-4529-a25e-ca904a7182c7.png)

Reply:
Ohh, looks like there already is a variant prop that we can edit in storybook. So maybe we can also make `variant` into a radio button list like this [comment](https://github.com/iTwin/iTwinUI-react/pull/872/files#r1004481871). Since variant is used in the `Folder` story too, might be better to update the global `argTypes` on [L28-32](https://github.com/iTwin/iTwinUI-react/blob/main/apps/storybook/src/Tile.stories.tsx#L28-L32) instead of the local one.

Reply:
Added argType to variant in Status, Loading and Custom Loading Icon stories. It made sense to have it only there.

Reply:
Actually, looks like `variant` is in the global `args` on [L34](https://github.com/iTwin/iTwinUI-react/blob/main/apps/storybook/src/Tile.stories.tsx#L34). So would be better to add the radio button lines you added like [L316](https://github.com/iTwin/iTwinUI-react/pull/872/files#diff-198e29f0f82eaa85da799271dd3ea518d23c0672cdbabc74f6f41c24e73c5b77R316) to the global `argTypes` on [L28-32](https://github.com/iTwin/iTwinUI-react/blob/main/apps/storybook/src/Tile.stories.tsx#L28-L32) instead of the three local stories